doc/user/build-install.xml

```diff
@@ -252,195 +277,88 @@ Python 3.7.1
     or <application>WinZip</application> on Windows.
     This will create a directory called
     <filename>scons-&buildversion;</filename>,
-    usually in your local directory.
-    Then change your working directory to that directory
-    and install &SCons; by executing the following commands:
+    usually in your local directory.  The driver script
+    will be in a subdirectory named <filename>scripts</filename>,
```

Comment:
Isn't it caps `Scripts` on windows?

Reply:
Yes.  Too many choices....

Reply:
> Yes. Too many choices....

actually, no.  It's Scripts on windows for an installed package, but this was talking about one of our tarballs, or a git checkout.

Reply:
gotcha.. should've expanding the compressed bit in the diff..